modules/ROOT/pages/dataweave-formats-avro.adoc

```diff
@@ -129,11 +129,13 @@ This format accepts properties that provide instructions for writing output data
 [cols="1,1,1,3a", options="header"]
 |===
 |Parameter |Type |Default|Description
-|`bufferSize`  |`Number`|`8192`|Size of the buffer writer.
-|`deferred`  |`Boolean`|`false`|Generates the output as a data stream when set to `true`, and defers the script's execution until consumed.
+|`bufferSize`  |`Number`|`8192`|Size of the buffer writer. The value must be greater than 8.
+
+|`deferred`  |`Boolean`|`false`|Generates the output as a data stream when set to `true`, and defers the script's execution until the generated content is consumed.
+
 
 Valid values are `true` or `false`.
-|`schemaUrl` (Required) |`String`|`null`|The URL for the Avro schema. Valid URI schemes are `classpath://`, `file://`, or `http://`.
+|`schemaUrl` (Required) |`String`|`''`|The URL for the Avro schema. Valid values are `classpath://`, `file://`, or `http://`.
```

Comment:
schemaUrl

Reply:
added reminder to next PR on this https://github.com/mulesoft/docs-dataweave/pull/225/files